### PR TITLE
fix my scientific innacuracy

### PR DESCRIPTION
**Pull request by Arkanic**
how did i do this

### CODE REVIEW

**Comment by DeltaNedas** [comment]
literally broken

**Comment by Arkanic** [comment]
i like to think of myself as the scientific type but i seriously added this error into my description

**Comment by joshuaptfan** [comment]
"Used for... highly conductive" is incorrect grammar.

Instead, say "Mineralized carbon, highly conductive and used for ammunition."

**Comment by CA6** [comment]
,and is, **and is highly** 
from the guy who wanted to correct me, _to be fair i can barely english_

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ tutorial.launch = Once you reach a specific wave, you are able to[accent] launch
 item.copper.description = The most basic structural material. Used extensively in all types of blocks.
 item.lead.description = A basic starter material. Used extensively in electronics and liquid transportation blocks.
 item.metaglass.description = A super-tough glass compound. Extensively used for liquid distribution and storage.
-item.graphite.description = Mineralized carbon, used for ammunition and highly conductive.
+item.graphite.description = Mineralized carbon, highly conductive and used as ammunition.
```
Comment (CA6) [inline core/assets/bundles/bundle.properties]: item.graphite.description = Mineralized carbon is highly conductive and is often used as ammunition

Comment (Arkanic) [reply core/assets/bundles/bundle.properties]: why does this need a review

**Comment by Arkanic** [comment]
anuke help

**Comment by Arkanic** [comment]
anuke help

File: core/assets/bundles/bundle.properties (hunk 1 above)
Comment (CA6) [reply core/assets/bundles/bundle.properties]: Because English-language is trash

**Comment by Arkanic** [comment]
haha bye

**Comment by CA6** [comment]
Thb big mood

**Pull request closed without merging**